docs/CODEOWNERS

```
@@ -7,4 +7,4 @@
 # More details are here: https://help.github.com/articles/about-codeowners/
 
 # These owners will be the default owners for everything in the repo.
-*     @UlisesGascon @inigomarquinez
+*     @onebeyond/admins
```

```suggestion
*     @onebeyond/admins
*     @onebeyond/maintainers 
```
I will say that maybe the maintainers should be part of it too.  what do you think? Technically they should be admin level ownership, based on [the documentation](https://github.com/onebeyond/admin/blob/main/how-to/manage-teams.md#organic-teams)

---

Agree! I'll change it